### DISCUSSION TITLE
D.2.10. Identity Resolution - EKG/MM

### DISCUSSION THREAD

**Discussion by giscus** [EKGF/ekg-maturity#56] [General]
# D.2.10. Identity Resolution - EKG/MM

Maturity Model for the Enterprise Knowledge Graph (EKG)

https://maturity-model.ekgf.org/pillar/technology/capability-area/technology-execution/capability/identity-resolution/

**Comment by ElisaKendall** [comment]
There are a couple of ontologies that are in the process of being standardized at the OMG for representing identifiers.  We extend this in FIBO and other ontologies with the notion of a registration authority and registrar, including the registry in which the identifier is registered.  The library that contains the starting point ontologies for this (identifiers, contextual identifiers) is in a private OMG git repo, but I can provide the ontologies / access as needed.  The git repo is at: https://github.com/ObjectManagementGroup/MVF, and the ontologies are under the Commons folder.  Extensions for registration authorities are in FIBO, and we can provide links to that if people are interested.  I'm hoping that over the next several months we will add registration authorities to the Commons Library.

**Comment by jgeluk** [comment]
See also https://ekgf.slack.com/archives/C03CYD797U0
